release_notes/3.2.2.md

```diff
@@ -1,6 +1,7 @@
-**Jira Release Notes - Published by Splunk December 22, 2021**
+**Jira Release Notes - Published by Splunk December 23, 2021**
 
 
-**Version 3.2.2 - Released December 22, 2021**
+**Version 3.2.2 - Released December 23, 2021**
```

Comment:
Can we update app_version in all places (File names and files) as it will be auto-increment after this PR will be merged

Reply:
done